src/commands/deploy/buildImageInAzure/BuildFromProjectListStep.ts

```diff
@@ -0,0 +1,60 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.md in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import { AzureWizardExecuteStep, AzureWizardPromptStep, IAzureQuickPickItem, IWizardOptions } from "@microsoft/vscode-azext-utils";
+import { localize } from "../../../utils/localize";
+import { IDeployBaseContext } from "../IDeployBaseContext";
+import { AcrListStep } from "../deployFromRegistry/acr/AcrListStep";
+import { BuildImageStep } from "./BuildImageStep";
+import { DockerFileItemStep } from "./DockerFileItemStep";
+import { IBuildImageInAzureContext } from "./IBuildImageInAzureContext";
+import { ImageNameStep } from "./ImageNameStep";
+import { OSPickStep } from "./OSPickStep";
+import { RootFolderStep } from "./RootFolderStep";
+import { RunStep } from "./RunStep";
+import { TarFileStep } from "./TarFileStep";
+import { UploadSourceCodeStep } from "./UploadSourceCodeStep";
+
+const buildImageInAzure = 'buildImageInAzure';
+const buildFromProjectLabels: string[] = [
+    localize('azure', 'Build from project remotely using Azure Container Registry')
+    //localize('docker', 'Build from project locally using Docker')
```

Comment:
I probably missed something, but what are we commenting this out until it's ready somewhere else?

Reply:
This is related to another create CA scenario (#285). This scenario won't be in this release so just putting a placeholder for now.